src/editorial/web/components/quote-icon/index.tsx

```diff
@@ -0,0 +1,121 @@
+import { css } from '@emotion/react';
+
+import { Format, Pillar, Special } from '@guardian/types';
+import { SvgQuote } from '@guardian/src-icons';
+import {
+	culture,
+	labs,
+	lifestyle,
+	news,
+	opinion,
+	specialReport,
+	sport,
+} from '@guardian/src-foundations';
+
+const quoteColor = (format: Format) => {
+	switch (format.theme) {
+		case Pillar.News: {
+			return css`
+				svg {
+					fill: ${news[400]};
+				}
+			`;
+		}
+		case Pillar.Opinion: {
+			return css`
+				svg {
+					fill: ${opinion[400]};
+				}
+			`;
+		}
+		case Pillar.Culture: {
+			return css`
+				svg {
+					fill: ${culture[400]};
+				}
+			`;
+		}
+		case Pillar.Sport: {
+			return css`
+				svg {
+					fill: ${sport[400]};
+				}
+			`;
+		}
+		case Pillar.Lifestyle: {
+			return css`
+				svg {
+					fill: ${lifestyle[400]};
+				}
+			`;
+		}
+		case Special.SpecialReport: {
+			return css`
+				svg {
+					fill: ${specialReport[400]};
+				}
+			`;
+		}
+		case Special.Labs: {
+			return css`
+				svg {
+					fill: ${labs[400]};
+				}
+			`;
+		}
+	}
+};
+
+type HeadlineSize = 'tiny' | 'small' | 'medium' | 'large';
```

Comment:
A point on conventions. Source uses `xsmall` as the next size down from `small` in places like [typography](https://theguardian.design/2a1e5182b/p/95d5d0-code/t/858844) and the [Button component](https://theguardian.design/2a1e5182b/p/435225-button/b/86f344/t/59cc6b). I wonder if in the name of consistency and predictability, we should continue this convention here?

Reply:
`teensyweensy`? 🙏

Reply:
Just in case there's no agreement on teensyweensy I pushed a change to `xsmall`